orthopairs/src/main/java/org/reactome/release/orthopairs/Main.java

```diff
@@ -48,12 +48,18 @@ public static void main( String[] args ) throws IOException, ParseException {
         // If using an alternative source species, specify the 4-letter code as the second argument
         //TODO: Better solution for this. What if user wants default config but to change source species?
         if (args.length > 0) {
-            pathToConfig = args[0];
-            sourceMappingSpecies = args[1];
+            if (args.length > 1) {
+                pathToConfig = args[0];
+                sourceMappingSpecies = args[1];
+            } else {
+                pathToConfig = args[0];
+                sourceMappingSpecies = "hsap";
+            }
         } else {
             pathToConfig = "src/main/resources/config.properties";
             sourceMappingSpecies = "hsap";
         }
```

Comment:
This could be replaced with:
```
String pathToConfig = args.length > 0 ? args[0] : Paths.get("src/main/resources/config.properties").toString()`
String sourceMappingSpecies = args.length > 1 ? args[1] : "hsap";
```

Reply:
Thanks! I'd been having trouble thinking of a succinct way to write these and I was just spacing on the ternary operators